src/test/java/com/razorpay/InvoiceClientTest.java

```diff
@@ -0,0 +1,200 @@
+        try {
+            mockResponseFromExternalClient(mockedResponseJson);
+            mockResponseHTTPCodeFromExternalClient(200);
+            Invoice invoice = invoiceClient.edit(INVOICE_ID,request);
+            assertNotNull(invoice);
+            assertEquals(INVOICE_ID,invoice.get("id"));
+            assertEquals("invoice",invoice.get("entity"));
+            assertTrue(invoice.has("receipt"));
```

Comment:
Add 1-2 more assert which assert the value also assertEquals(expectedValue, actualValue)

Reply:
i have added that in all functions can you please check